sdcm/utils/nemesis_utils/node_operations.py

```diff
@@ -48,4 +76,6 @@ def is_node_removed_from_cluster(removed_node: BaseNode, verification_node: Base
 
 def is_node_seen_as_down(down_node: BaseNode, verification_node: BaseNode) -> bool:
     LOGGER.debug("Verification node %s", verification_node.name)
-    return down_node not in verification_node.parent_cluster.get_nodes_up_and_normal(verification_node)
+    nodes_status = verification_node.parent_cluster.get_nodetool_status(verification_node, dc_aware=False)
+    down_node_status = nodes_status.get(down_node.ip_address)
```

Comment:
shouln't use `down_node.listen_address`?

Reply:
It will be the same as ip_address and we use it everywhere with nodetool_status